src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -20,17 +20,22 @@
           {{getIdentifier}}</strong> back on the register.
         </p>
 
+        <p v-if="isAdministerFreeze">
+          You are about to {{ !isAdminFreeze ? 'Freeze' : 'Unfreeze' }} <span class="text-uppercase font-weight-bold">{{getEntityName}}</span>,
```

Comment:
This line is longer than the 120 limit. If possible, keep it about the same length as the other lines in this component.

![image](https://user-images.githubusercontent.com/10002889/212379168-cead05d5-3e88-4942-a73c-5d2754d5f612.png)